### DISCUSSION TITLE
algerian channel

### DISCUSSION THREAD

**Discussion by jaouadledz** [iptv-org/community#238] [Show and tell]
Hello, this is to tell you that no Algerian channel works can solve the problem please.
cordially

**Comment by Sphinxroot** [comment]
the problem  all channel that is hosted on https://cdn02.hta.dz/  now need the base 64 token to play

as exemple  Canal Algerie
https://cdn02.hta.dz/abr_htatv/CANAL_ALGERIE/playlist.m3u8??wmsAuthSign=c2VydmVyX3RpbWU9MDgvMjMvMjAyMSAwMjozMjozMSBBTSZoYXNoX3ZhbHVlPU02Qmp0TGtxT2hac2dXQmh3NmtEL1E9PSZ2YWxpZG1pbnV0ZXM9MTA=

base 64 decoded = server_time=08/23/2021 02:32:31 AM&hash_value=M6BjtLkqOhZsgWBhw6kD/Q==&validminutes=10

so every 10min the token is changed

**Comment by Dum4G** [comment]
@LaneSh4d0w can we have your take on this one?

**Reply by BellezaEmporium** [reply]
https://github.com/iptv-org/iptv/issues/1961#issuecomment-904746212

**Reply by BellezaEmporium** [reply]
Please test using https://iptv-all.lanesh4d0w.codes/algeria/hta_all_in_1?channel= *insert channel name* . Note that YOU MUST connect using this proxy : (Socks4) 190.242.128.35 | 5678 to get the different channels.

P.S = Don't ask why I've used a Colombian proxy once again, they seem to be the most stable proxies I have for now. French proxies didn't connect, and the availability of Algerian proxies are mediocre.

EDIT : Exactly what I've expected : 

![image](https://user-images.githubusercontent.com/30985701/131219908-8f430960-f7e7-4707-85ed-4ffca8ffeeba.png)

**Reply by BellezaEmporium** [reply]
wait... has anyone tried "#EXTVLCOPT:https-proxy=socks4://190.242.128.35:5678" ?

**Reply by Dum4G** [reply]
I can give it a try but this proxy will face the same treat as linkastream and your query solution. This is not a permanent fix

**Reply by BellezaEmporium** [reply]
> 
> 
> I can give it a try but this proxy will face the same treat as linkastream and your query solution. This is not a permanent fix

Can't do more that what I can actually do mate, sorry. I know this isn't permanent but at least this'll do the trick while the Algerian govt. get their channels back.

**Reply by Dum4G** [reply]
Tried SamiraTV, SamiraTV_240p, TV_4 with #EXTVLCOPT:https-proxy=socks4://190.242.128.35:5678. No luck

**Reply by BellezaEmporium** [reply]
Oh okay, at least we've tried.
